docker/mongo/a_init_replicas.js

```diff
@@ -0,0 +1,33 @@
+
+/*
+
+This script is the first of two scripts responsible for setting up mongoDB on initial boot. 
+These scripts should be copied to the /docker-entrypoint-initdb.d/ directory within the docker image;
+the docker image will then execute these scripts automatically when the database is first created. 
+This script and is partner are prefixed with the letters a and b respectively to ensure they are run 
```

Comment:
Should this be "This script and **its** partner"?

Reply:
updated!